src/test/llm/llmnode_test.cpp

```diff
@@ -190,39 +205,59 @@ TEST_F(LLMFlowHttpTest, unaryCompletionsJson) {
-        ASSERT_FALSE(choice["logprobs"].IsObject());
-        ASSERT_TRUE(choice["text"].IsString());
-        EXPECT_STREQ(choice["text"].GetString(), expectedMessages[i].c_str());
-        ASSERT_EQ(choice["index"], i++);
-    }
+    // TODO: In the next step we should break this suite into smaller ones, use proper configuration instead of such if...else...
+    if (params.modelName.find("vlm") == std::string::npos) {
+        ASSERT_EQ(
+            handler->dispatchToProcessor(endpointCompletions, requestBody, &response, comp, responseComponents, writer),
+            ovms::StatusCode::OK);
+        parsedResponse.Parse(response.c_str());
+        ASSERT_TRUE(parsedResponse["choices"].IsArray());
+        ASSERT_EQ(parsedResponse["choices"].Capacity(), 1);
+        int i = 0;
+        for (auto& choice : parsedResponse["choices"].GetArray()) {
+            ASSERT_TRUE(choice["finish_reason"].IsString());
+            ASSERT_FALSE(choice["logprobs"].IsObject());
+            ASSERT_TRUE(choice["text"].IsString());
+            if (params.generateExpectedOutput) {
+                EXPECT_STREQ(choice["text"].GetString(), expectedMessages[i].c_str());
+            }
+            ASSERT_EQ(choice["index"], i++);
+        }
 
-    ASSERT_TRUE(parsedResponse["usage"].IsObject());
-    ASSERT_TRUE(parsedResponse["usage"].GetObject()["prompt_tokens"].IsInt());
-    ASSERT_TRUE(parsedResponse["usage"].GetObject()["completion_tokens"].IsInt());
-    ASSERT_TRUE(parsedResponse["usage"].GetObject()["total_tokens"].IsInt());
-    ASSERT_EQ(parsedResponse["usage"].GetObject()["completion_tokens"].GetInt(), 5 /* max_tokens */);
-    EXPECT_STREQ(parsedResponse["model"].GetString(), "llmDummyKFS");
-    EXPECT_STREQ(parsedResponse["object"].GetString(), "text_completion");
+        ASSERT_TRUE(parsedResponse["usage"].IsObject());
+        ASSERT_TRUE(parsedResponse["usage"].GetObject()["prompt_tokens"].IsInt());
+        ASSERT_TRUE(parsedResponse["usage"].GetObject()["completion_tokens"].IsInt());
+        ASSERT_TRUE(parsedResponse["usage"].GetObject()["total_tokens"].IsInt());
+        ASSERT_EQ(parsedResponse["usage"].GetObject()["completion_tokens"].GetInt(), 5 /* max_tokens */);
+        EXPECT_STREQ(parsedResponse["model"].GetString(), params.modelName.c_str());
+        EXPECT_STREQ(parsedResponse["object"].GetString(), "text_completion");
+    } else {  // Completions endpoint not supported for VLM servable
+        ASSERT_EQ(
+            handler->dispatchToProcessor(endpointCompletions, requestBody, &response, comp, responseComponents, writer),
+            ovms::StatusCode::MEDIAPIPE_EXECUTION_ERROR);
+    }
 }
 
-TEST_F(LLMFlowHttpTest, unaryCompletionsJsonSpeculativeDecoding) {
+TEST_P(LLMFlowHttpTestParameterized, unaryCompletionsJsonSpeculativeDecoding) {
+    auto params = GetParam();
+    // TODO: In the next step we should break this suite into smaller ones, use proper configuration instead of skipping
```

Comment:
which jira is the next step?

Reply:
https://jira.devtools.intel.com/browse/CVS-164532